azure-pipelines.yml

```diff
@@ -7,32 +7,25 @@ trigger:
 - master
 
 pool:
-  vmImage: 'Ubuntu-16.04'
+   vmImage: 'ubuntu-latest'
```

Comment:
Would it make sense to keep pinning the version?

Reply:
it did not help this time, so I'm guessing no